README.md

```diff
@@ -87,12 +102,12 @@ When you access a color by name (i.e. `colors.indigo`), you'll find a palette of
 **CSS-in-JS**
 
 ```jsx
-import styled from '@emotion/styled';
-import {colors} from '@apollo/space-kit';
+import styled from "@emotion/styled";
+import { colors } from "@apollo/space-kit";
```

Comment:
are there docs on how to import / use the icons? I have tried with the beta version and am not able to render anything by doing 

`import { IconBack } from `@apollo/space-kit/icons/IconBack`;` and then `<IconBack />`

Reply:
There's no fill issue, you're not setting a size so the elements are rendering at 0px wide and 0px tall. I apparently left this out of the readme. Maybe we should also have an example and a FAQ like "why isn't anything rendering?" and "how do I import icons"

Reply:
oo thank you! If you want notes on what I did to debug I can keep some.